src/components/search/GlobalSearchField.js

```diff
@@ -6,24 +6,28 @@
 
```

Comment:
There is a known issue. There are console warning:
`warning: validateDOMNesting(...): <li> cannot appear as a descendant of <li>.
    in li (created by ForwardRef(ListItem))
    in ForwardRef(ListItem) (created by WithStyles(ForwardRef(ListItem)))
    in WithStyles(ForwardRef(ListItem)) (at GlobalSearchField.js:118)
    in ForwardRef (at GlobalSearchField.js:181)`